app/assets/stylesheets/image_x.scss

```diff
@@ -71,3 +77,21 @@ $image-thumb-slider-border-color: #000;
 .#{$namespace}-image-x-label {
   text-align: center;
 }
+
+.frame {
+    position:absolute;
+    display:block;
+    border: 1px solid black;
+}
+.canvas {
+    position: absolute;
+    display: block;
+    border: 1px solid black;
+}
+.frame h4 {
+    width:100%;
+    height:35px;
+    position:absolute;
+    bottom:0;
+    text-align:center;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Colon after property should be followed by one space